config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: infrastructure-manager
-  newTag: 0.0.1
+  newName: europe-docker.pkg.dev/kyma-project/dev/infrastructure-manager
+  newTag: PR-42
```

Comment:
I don't yet understand fully `kustomize` and I wonder if this should stay like it is after following change 60ba882516136e1642bdba83a2113c524b16744e

Reply:
This change can be reverted as well.